eda_analysis/eda_analysis.py

```diff
@@ -78,8 +85,50 @@ def describe_cat_var(dataframe,cat_vars):
     >>> describe_cat_variable(X,cat_vars)
        
     """
-    # Code 
     
+    #Checking for valid inputs
```

Comment:
Should we also test whether numeric columns are not selected? Although there wouldn't be an error, the plot looks not right when numeric columns are selected.

Reply:
Even numeric columns can be categorical. Suppose a column has only 3 types of values (1,2,3) then it would be numeric yet categorical so I didn't put such constraints.

Reply:
Yes, we can leave that as it is.